Packs/CiscoAMP/Integrations/CiscoAMP/CiscoAMP_description.md

```diff
@@ -0,0 +1,46 @@
+# Cisco AMP Secure Endpoint
+Cisco Advanced malware protection software is designed to prevent, detect, and help remove threats in an efficient manner from computer systems.
+Threats can take the form of software viruses and other malware such as ransomware, worms, Trojans, spyware, adware, and fileless malware.
+
+# See the API Documentation
+The API documentation can be found in: [Cisco AMP API](https://api-docs.amp.cisco.com/).</br>
+Choose the relevant API and then select version: `v1`.
+
+# Fetch Incidents
+Incidents are fetched through the command: `cisco-amp-event-list`.</br>
+The fetched event types can be controlled through `event_id` that can be received from the command: `cisco-amp-event-type-list`.
+
+# Polling Command
+The commands: `cisco-amp-computer-isolation-create` and `cisco-amp-computer-isolation-delete` supports polling.</br> The polling is done when the status of an endpoint has changed.
+
+# Integration Commands

```

Comment:
Is there any reason to list the Integration commands? I don't think that it is necessary and recommend that they be removed.

Reply:
I removed `# Integration Commands` and all that was below as requested.